server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java

```diff
@@ -268,6 +279,9 @@ public IRateSource createRateSource(String sourceLogin) {
                     preferredFiatCurrency = st.nextToken().toUpperCase();
                 }
                 return new CoingiExchange(preferredFiatCurrency);
+            }else if ("lunoRateSource".equalsIgnoreCase(rsType)) {
+                String preferedFiatCurrency = FiatCurrency.ZAR.getCode();
```

Comment:
Not sure if calculation from ZAR to other currencies like USD will work. But if ATMs will run in ZAR it will work.